snap/local/patches/0001-optimize-build-for-pipeline-CI-check.patch

```diff
@@ -107,8 +107,8 @@ index 1e537489..faf42425 100644
    security-secretstore-setup:
      adapter: full
      after: [vault]
-@@ -271,29 +202,6 @@ apps:
-       ExecutorPath: $SNAP/bin/sys-mgmt-agent-snap-executor.sh
+@@ -277,29 +205,6 @@ apps:
+       EXECUTORPATH: $SNAP/bin/sys-mgmt-agent-snap-executor.sh
```

Comment:
Because it had already been updated in `snapcraft.yaml` (by Lenny), and by default, the patch command will apply a fuzz of 1, meaning if the patch doesn't apply the first time, it'll go back and try to re-apply ignoring the first and last line of the context/hunk, which in this case includes the `ExecutorPath/EXECUTORPATH` line